### DISCUSSION TITLE
is there an active open source project using nx

### DISCUSSION THREAD

**Discussion by dagda1** [nrwl/nx#7761] [General]
Is there a good open source project that uses nx?

I've had a look at storybook and although they have the nx.json file, they are still using lerna.

Even this repo seems to be using it sparingly but I am very happy to be proved wrong on this one.

**Comment by brandonroberts** [comment]
Yep, there are a few actually.

https://github.com/ngrx/platform
https://github.com/NativeScript/NativeScript
https://github.com/valor-software/ngx-bootstrap
https://github.com/angular-eslint/angular-eslint

It's not uncommon to use nx and Lerna together with Lerna just being used for versioning/publishing, and Nx for everything else.
